src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -29,31 +29,40 @@ export const CentralImportDuplicateSearch = ({ duplicateSearchFilters }: Central
   duplicateSearchFilters.yearPublished.length > 0 &&
     queryArray.push(`${DescriptionFieldNames.PublicationYear}:"${duplicateSearchFilters.yearPublished}"`);
 
-  const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')})` : '';
+  const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')}&results=${maxHits})` : '';
 
-  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
-    url: `${SearchApiPath.Registrations}?${searchQuery}&results=${maxHits}`,
-    errorMessage: t('feedback.error.search'),
+  const importCandidateQuery = useQuery({
+    queryKey: ['importCandidates', searchQuery],
+    queryFn: () => fetchImportCandidates(),
+    onError: () =>
+      dispatch(
+        setNotification({
+          message: t('feedback.error.get_messages'),
+          variant: 'error',
+        })
+      ),
   });
 
+  const searchResults = importCandidateQuery.data?.hits ?? [];
+
   return (
     <>
-      {isLoadingSearchResults ? (
+      {importCandidateQuery.isLoading ? (
         <ListSkeleton minWidth={100} maxWidth={100} height={100} />
       ) : (
         searchResults && (
           <>
             <Typography variant="subtitle1">
               {t('basic_data.central_import.duplicate_search_hits_shown', {
-                ShownResultsCount: searchResults.hits.length,
-                TotalResultsCount: searchResults.size,
+                ShownResultsCount: searchResults.length,
+                TotalResultsCount: searchResults,
```

Comment:
Antar vi fortsatt vil at denne skal vise totalt antall uavhengig av paginering?
```suggestion
                TotalResultsCount: importCandidateQuery.data.size,
```